package.json

```diff
@@ -1,20 +1,21 @@
 {
   "name": "topbar",
-  "main": "topbar.min.js",
   "files": [
     "topbar.js",
     "topbar.min.js",
-    "*.d.ts"
+    "*.d.ts",
+    "dist"
   ],
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
I think a patch version is fine here. If I did it ok previous behaviour should still work. I tried documentation page and it keeps working as a script. 

The difference is that now people can import as ESM module

```javascript
// ./src/app.ts
import { Topbar } from 'topbar' // ? I don't think is jet published

declare global {
  interface Window {
    topbar: Topbar
  }
}

window.topbar = new Topbar({
  barColors: {
    0: 'rgba(255, 153, 153, .9)',
    '.17': 'rgba(255, 204, 153, .9)',
    '.34': 'rgba(255, 255, 153, .9)',
    '.51': 'rgba(153, 255, 153, .9)',
    '.68': 'rgba(153, 204, 255, .9)',
    '.85': 'rgba(204, 153, 255, .9)',
    '1.0': 'rgba(204, 153, 255, .9)',
  },
  shadowColor: 'rgba(0, 0, 0, .1)',
})
```

Reply:
I'm just watching this project but here's my two cents.

IMO, a minor version bump is more appropriate for this PR. Bumping the patch version is reserved just for patches (bug fixes). Since this PR isn't fixing bugs but rather adding new features, a minor bump would be the better way to go.

Reply:
Agree, true